lib/ups/version.rb

```diff
@@ -1,7 +1,7 @@
 module UPS
   module Version
     MAJOR = 0
-    MINOR = 8
+    MINOR = 9
```

Comment:
One more thing, let's bump the version in a standalone commit, on master just before releasing, because we might have multiple PRs open at once and if they all modify the version it will become difficult during merging.

Reply:
@tehpeh yep I agree, I'll revert that and merge this